packages/charting/src/components/DonutChart/Arc/Arc.styles.ts

```diff
@@ -2,6 +2,14 @@ import { IArcProps, IArcStyles } from './Arc.types';
 export const getStyles = (props: IArcProps): IArcStyles => {
   const { color } = props;
   return {
-    root: { fill: color }
+    root: { fill: color },
+    hover: {
+      height: '76px',
+      width: '143px',
+      fontSize: '28px',
+      color: '#0078D7',
```

Comment:
The color should come from a `theme` prop rather than being hardcoded, please see other charting components for examples.